crates/parse/src/parser/mod.rs

```diff
@@ -733,7 +733,6 @@ impl<'sess, 'ast> Parser<'sess, 'ast> {
         }
         self.prev_token = std::mem::replace(&mut self.token, next);
         self.expected_tokens.clear();
-        self.docs.clear();
```

Comment:
I don't removing this is right, if parse_docs is not called then docs are accumulated to the next item that does

```
/// a
x;
/// b
uint y = 0;
```

the statement will get the `a` doc when it shouldnt